woof-code/rootfs-skeleton/usr/bin/service2initd

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+#Converts systemd .service file into init.d script
+#Written by mistfire
+
+SRVFILE="$1"
+
+if [ "$SRVFILE" == "" ] || [ ! -e "$SRVFILE" ]; then
+ exit
+fi
+
+SRVNAME="$(basename $SRVFILE .service)"
+
+if [ "$(cat $SRVFILE | grep -E '^Type\ =|^Type=' | grep -E 'dbus|udev')"  == "" ]; then
+	 
+	 SRVTYPE="$(cat $SRVFILE | grep '^Type=' | cut -f 2 -d '=' | xargs)"
+	 
+	 EXECCMD="$(cat $SRVFILE | grep '^ExecStart=' | sed -e 's#^ExecStart=##' | xargs)"
+	 PREEXEC="$(cat $SRVFILE | grep '^ExecStartPre=' | sed -e 's#^ExecStartPre=##' | xargs)"
+	 POSTEXEC="$(cat $SRVFILE | grep '^ExecStartPost=' | sed -e 's#^ExecStartPost=##' | xargs)"
+	 EXECSTOP="$(cat $SRVFILE | grep '^ExecStop=' | sed -e 's#^ExecStop=##' | xargs)"
+	 POSTEXECSTOP="$(cat $SRVFILE | grep '^ExecStopPost=' | sed -e 's#^ExecStopPost=##' | xargs)"
+	 RELOADEXEC="$(cat $SRVFILE | grep '^ExecReload=' | sed -e 's#^ExecReload=##' | xargs)"
+
+	 PIDFILE="$(cat $SRVFILE | grep '^PIDFile=' | cut -f 2 -d '=' | xargs)"
+	 KILLSIG="$(cat $SRVFILE | grep '^KillSignal=' | cut -f 2 -d '=' | xargs)"
+	 WKDIR="$(cat $SRVFILE | grep '^WorkingDirectory=' | cut -f 2 -d '=' | xargs)"
```

Comment:
This list of fields is partial - for example, Restart= is not taken care of and doesn't translate well to sysvinit